QuantConnect.TradeStationBrokerage/Api/TradeStationApiClient.cs

```diff
@@ -225,44 +225,45 @@ public async Task<bool> CancelOrder(string orderID)
                 tradeStationOrder.StopPrice = stopPrice.Value.ToStringInvariant();
                 break;
         }
-        Log.Debug($"{nameof(TradeStationApiClient)}.{nameof(PlaceOrder)}.Body: {JsonConvert.SerializeObject(tradeStationOrder, jsonSerializerSettings)}");
         return await RequestAsync<TradeStationPlaceOrderResponse>(_baseUrl, $"/v3/orderexecution/orders", HttpMethod.Post,
             JsonConvert.SerializeObject(tradeStationOrder, jsonSerializerSettings));
     }
 
     /// <summary>
-    /// Replaces an existing order in TradeStation with the provided Lean order.
+    /// Replaces an existing order with the specified parameters.
     /// </summary>
-    /// <param name="order">The Lean order to replace the existing order.</param>
-    /// <returns>The response containing the result of the order replacement.</returns>
-    public async Task<Models.OrderResponse> ReplaceOrder(Lean.Order order)
+    /// <param name="brokerId">The unique identifier of the broker.</param>
+    /// <param name="leanOrderType">The type of order to be placed.</param>
+    /// <param name="newQuantity">The new quantity for the order. If null, the quantity remains unchanged.</param>
+    /// <param name="limitPrice">The limit price for the order. If null, the limit price remains unchanged.</param>
+    /// <param name="stopPrice">The stop price for the order. If null, the stop price remains unchanged.</param>
+    /// <returns>A task representing the asynchronous operation. The task result contains an <see cref="OrderResponse"/> object representing the response to the order replacement.</returns>
+    /// <remarks>
+    /// This method replaces an existing order with new parameters such as quantity, limit price, and stop price.
+    /// If any parameter is not provided (null), the corresponding value of the existing order will remain unchanged.
+    /// </remarks>
+    public async Task<Models.OrderResponse> ReplaceOrder(TradeStationAccountType accountType, string brokerId, OrderType leanOrderType, decimal newQuantity, decimal? limitPrice = null, decimal? stopPrice = null)
     {
-        var orderID = order.BrokerId.Single();
+        var accountID = (await GetAccounts()).Single(acc => acc.AccountType == accountType).AccountID;
```

Comment:
Get the account id once at the start, constructor, initialize, and keep it, it shouldn't change